ignition/state_config.json

```diff
@@ -14,7 +14,7 @@
43f02a5d47b01d5d4bb01e340104521af40000920003101af96000980800009700007f4af800009500003f960800001aec5000910000f01a43a0001a47e0005d4bf03d104920c05d4ff03e104d30c05053b03072540020285125405fec000a504940287254002028493540504bb0a8724c0048284944c0504fb0781ae920001ae5300020f8330058fbe00250fbe0047500049b1a4bd000724c003028ed24c0724800302843b4801af50000920000f01af91000980800009700003f4af800009500007f960800001aec5000910002c01a43a0001a5be0005d47f03f104510c05d4bf040104920c0504fb02072500020284d15005fec00085045302872500020284525005047b15072480048284534801ae9100020f8330058fbe00250fbe004750006fd5047b0685fec000d504bb19872500030284905005053b1c872540030285115401ae920001ae5400020f8330058fbe00250fbe004750005531a47d0001345100013451000764400445047b2407248004828453480504bb288724c0030284904c01ae910001ae5200020f8330058fbe00250fbe0047500024b5047b09872480030284504805043b0e81ae9000020f8330058fbe00250fbe004750007701a43d0005d4900005d4d00015d4100025053b1f872540030285115405047b2285fed20455fed30465fed00475043b1101ae940001ae510001ae1000020f8330058fbe00250fbe004750003541a43d0005d4500005d4900015d410002504fb1385fed10275fed20285fed00295043b1001ae930001ae5000020f8330058fbe00250fbe004750007771a43d0005047b01072480010284504805043b0d872480010284114805d43f02b5d47b01b5d4bb01c340104521af40000740000155043b2b85fec00575047b1281ae900001ae5100020f8330058fbe00250fbe0047500023c1a43d0007244001028ed04405043b0c8724400102843b4405d43f02c5d47b0195d4bb01a340104525d43f02336400000920002c01af96000980800009700007f4af800009500001f960800001aec5000910001101a43e0005d47f041104510c0504bb0681ae9200020f8330058fbe00250fbe0047500090c1a4bd000724c002828ed24c0504bb0485fed10097244001b5fed100a5047b058724c0010284524c0504bb0d8724c0028284bb4c0504fb10072500010284d15005047b0981ae920001ae530001ae1100020f8330058fbe00250fbe004750001e31a47d000504bb028724c0020284914c05047b0b8724c0020284524c0504bb0901ae910001ae5200020f8330058fbe00250fbe0047500020b1af40000920001101af90000980800009700001f4af800004700000000000000000000010000000000000000fc96a3a99ae1873e9e571a8be7d14111a2b4b7bd3abacb367c6e0f79c9c149d90000000000000000000000000000000000000000000000000000000000000000000000000000000070726f78795f746172676574000000007365745f70726f78795f74617267657470726f78795f6f776e65720000000000696e697469616c697a655f70726f787935fa5b7532d53cf687e13e3db014eaf208c5b8c534ab693dd7090d5e02675f3e7bb458adc1d118713319a5baa00a2d049dd64d2916477d2688d76970c898cd55b18f98389493a4aa04e99cbbb64c21d6e36d6f4cedecaa693fbaeea0cf660e8a7365745f70726f78795f6f776e657200cccccccccccc00021dfe7feadc1d9667ffffffffffff00003f702ea3351c9c1e46616c6c6261636b4572726f723a3a5461726765744e6f7453657400000000008c25cb3686462e9a1ddc0adda1270a0196dd838b44f99d8c3c6e90ae504df6aa00000000000034cc000000000000318c00000000000030d40000000000002f440000000000002ef00000000000002ee80000000000002e680000000000002e800000000000002e380000000000002e300000000000002de000000000000025880000000000000a440000000000000614000000000000060c0000000000000624000000000000042000000000000004180000000000000388000000000000038000000000000001f8",
```

Comment:
I updated this bytecode with forc 0.63.3 which includes some security fixes just to be extra safe